### DISCUSSION TITLE
Make command not working

### DISCUSSION THREAD

**Discussion by Atvee** [Cyfrin/security-and-auditing-full-course-s23#239] [Q&A]
when I try to run the make command for the puppy raffle audit, I am getting the following output in my terminal

<img width="909" alt="Screenshot 2024-10-02 at 2 25 02 PM" src="https://github.com/user-attachments/assets/1bf4e3cb-48ff-4eee-a4b5-e1786d2be6cc">

Can anyone please help me with this

**Comment by EngrPips** [comment]
What make command did you run?

**Reply by Atvee** [reply]
I was following along with patrick through the course and ran the make command specified in the makefile of the lesson
I was able to solve the issue as it was some problem with my file path and ran the make command again, but now I encounter another problem

<img width="754" alt="Screenshot 2024-10-02 at 2 59 52 PM" src="https://github.com/user-attachments/assets/6856f3f6-1d2a-42b5-893a-8925f4c6fe67">
 
I tried installing homebrew and install the specific solidity version 0.7.6 but is still encountering the issue

**Reply by Atvee** [reply]
Seems like solc is for intel based systems and doesn't work for apple silicon chips????
So what should I do?

**Reply by EngrPips** [reply]
I am unsure what this issue is. Let's hope that some giga Chad sees this and helps. You can also try to browse the internet maybe you will find a solution.

**Reply by Atvee** [reply]
Yeah I saw somewhere installing rosetta to emulate intel might work
I might try it , let's see

**Reply by EngrPips** [reply]
Looking forward to how that plays out.
